cloud: support generating endpoints automatically (#14244)
pingcap/docs: tidb-cloud/data-service-api-key.md

## Rate limiting

Request quotas are rate-limited as follows:

- 100 requests per minute (rpm) per API key
- 100 requests per day for each Chat2Query Data App

> If you exceed the rate limit, the API returns a `429` error. For an increase in quota, you can [submit a request](https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519) to our support team.

Reviewer: ```suggestion
If you exceed the rate limit, the API returns a `429` error. To increase your quota, you can [submit a request](https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519) to our support team.
```
